lib/rubocop/cop/lint/empty_conditional_body.rb

```diff
@@ -53,15 +59,36 @@ module Lint
       #   end
       #
       class EmptyConditionalBody < Base
+        extend AutoCorrector
         include CommentsHelp
+        include RangeHelp
 
         MSG = 'Avoid `%<keyword>s` branches without a body.'
 
         def on_if(node)
           return if node.body
           return if cop_config['AllowComments'] && contains_comments?(node)
 
-          add_offense(node, message: format(MSG, keyword: node.keyword))
+          add_offense(node, message: format(MSG, keyword: node.keyword)) do |corrector|
+            remove_case(corrector, node)
+            remove_comments(corrector, node)
```

Comment:
I wonder if utilizing `source_range_with_comment` won't allow you to remove that in a single method